staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2683,8 +2683,18 @@ type InterfaceBindingPlugin struct {
 	// If namespace is not specified, VMI namespace is assumed.
 	// version: 1alphav1
 	NetworkAttachmentDefinition string `json:"networkAttachmentDefinition,omitempty"`
+	// DomainAttachmentType is a standard domain definition that kubevirt supports.
+	// Supported values: "tap".
+	// The standard domain attachment can be used instead or in addition to the SidecarImage.
```

Comment:
nit: The formal API uses a non-capital `s`, i.e. `sidecarImage`. Sorry for confusing you in my prev comment.

If you do not need to send another patch, I would not bother with it.